src/Compilers/CSharp/Test/Semantic/Diagnostics/OperationAnalyzerTests.cs

```diff
@@ -211,7 +211,7 @@ public void M1()
                 );
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/8699")]
```

Comment:
Will unskip.

---
In reply to: [70328205](https://github.com/dotnet/roslyn/pull/12416#discussion_r70328205) [](ancestors = 70328205)